apps/web/modules/cache/redis.test.ts

```
@@ -0,0 +1,261 @@
+
+    // Make connect resolve successfully
+    mockRedisClient.connect.mockResolvedValue(undefined);
+  });
+
+  afterEach(() => {
+    vi.resetModules();
+    delete process.env.REDIS_URL;
+  });
+
+  describe("Module initialization", () => {
+    it("should create Redis client when REDIS_URL is set", async () => {
```

ESLint

[error] Prefer using test instead of it within describe

(@vitest/consistent-test-it)